docs/price-feed/index.adoc

```diff
@@ -7,10 +7,10 @@
 ifndef::tbtc[toc::[]]
 
 The price feed is an integral part of the system, ensuring sufficient
-collateral backs all tBTC signers. For tBTC v1, the feed will be built on the
-https://developer.makerdao.com/feeds/[USD price feeds from MakerDAO], currently
-operated by MakerDAO. MakerDAO has both ETHUSD and BTCUSD feeds; these are
-combined to make a single ETHBTC price.
+collateral backs all tBTC signers. For tBTC v1, the feed will be built on a
+BTCETH
+https://developer.makerdao.com/feeds/[https://docs.makerdao.com/smart-contract-modules/oracle-module/oracle-security-module-osm-detailed-documentation]
+from MakerDAO, currently operated by MakerDAO.
 
 The minimal price feed design is specified completely by the interface below:
 
```

Comment:
In this interface still accurate?

Reply:
Probably not, but that’s for another day.